### DISCUSSION TITLE
How do I tell yazi to set the terminal title to PWD

### DISCUSSION THREAD

**Discussion by UtkarshVerma** [sxyazi/yazi#671] [1. Q&A]
I like to have my terminal title updated whenever the file manager changes folders. In `lf`, I used to have this hook do it for me:

```
cmd on-cd &{{
    # '&' commands run silently in background (which is what we want here),
    # but are not connected to stdout.
    # To make sure our escape sequence still reaches stdout we pipe it to /dev/tty
    printf "\033]0;$(pwd | sed "s|$HOME|~|") - lf\007" >/dev/tty
}}
```

Do we have something similar for `yazi`. Btw, thanks for making this tool. I like how it has sensible defaults and how hackable it is.

**Comment by sxyazi** [comment]
Not supported atm, but we can add it

**Reply by UtkarshVerma** [reply]
Thanks, please take your time. I'll keep this discussion open until it is implemented.

**Reply by UtkarshVerma** [reply]
Thanks, please take your time. I'll keep this discussion open until it is implemented.

**Reply by sxyazi** [reply]
Would you or someone else be interested in raising a PR for it?

**Reply by UtkarshVerma** [reply]
I am afraid working up a PR would not be possible for me at the moment.

**Comment by sxyazi** [comment]
This is now supported, see https://yazi-rs.github.io/docs/configuration/yazi/#manager.title_format
